src/ui/component/fileListItem/view.jsx

```diff
@@ -0,0 +1,106 @@
+// @flow
+import React, { useEffect } from 'react';
+import classnames from 'classnames';
+import { convertToShareLink } from 'lbry-redux';
+import { withRouter } from 'react-router-dom';
+import { openCopyLinkMenu } from 'util/context-menu';
+import { formatLbryUriForWeb } from 'util/uri';
+import CardMedia from 'component/cardMedia';
+import UriIndicator from 'component/uriIndicator';
+import TruncatedText from 'component/common/truncated-text';
+import DateTime from 'component/dateTime';
+import FileProperties from 'component/fileProperties';
+
+type Props = {
+  uri: string,
+  claim: ?StreamClaim | ?ChannelClaim,
+  obscureNsfw: boolean,
+  claimIsMine: boolean,
+  pending?: boolean,
+  resolveUri: string => void,
+  isResolvingUri: boolean,
+  preventResolve: boolean,
+  history: { push: string => void },
+  thumbnail: string,
+  title: string,
+  nsfw: boolean,
```

Comment:
Out of scope, but in case you're doing any relevant refactoring, this is dying / should die and should instead be a set of `0...n` objectionable attributes, each of which the user can opt out of displaying individually.